ucm2/Intel/sof-essx8336/HiFi.conf

```diff
@@ -98,6 +60,16 @@ SectionDevice."Headphones" {
 	ConflictingDevice [
 		"Speaker"
 	]
+	
+	EnableSequence [
+                cset "name='Headset Mic Switch' on"
+                cset "name='Speaker Switch' off"
+                cset "name='Differential Mux' lin2-rin2"
```

Comment:
Changing controls which are evidently assigned to the input for the output device seems very suspicious. I would remove this addition. Also do not use controls which are already assigned to other devices (Speaker).